agent_deploy/kubernetes/sysdig-kmod-thin-agent-slim-daemonset.yaml

```diff
@@ -0,0 +1,135 @@
+### WARNING: this file is supported from Sysdig Agent 0.80.0
+# apiVersion: extensions/v1beta1  # If you are in Kubernetes version 1.8 or less please use this line instead of the following one
```

Comment:
Can we remove this line? I don't know that we need to support the kmodule-thin in 1.8 clusters. cc @balinaveen

Reply:
Sure, if there's no requirement to support k8s <= 1.8 this can be removed. Can we check with IBM ?

Reply:
I am taking leadership of this task. I am trying to answer here old comments, and I will prepare a new PR (as this one is based in a fork)
According to https://sysdig.atlassian.net/browse/SMAGENT-2906, kubernetes versions range from 1.17 to 1.20
So we can delete this line